### DISCUSSION TITLE
Feature request: Select which polyfills to implement

### DISCUSSION THREAD

**Discussion by jimmymorris** [chakra-ui/panda#2510] [Ideas]
I understand we can handle the polyfills through postCSS but if Panda is going to ship polyfills out of the book, I think it would make sense to allow cherry picking of those polyfills.

The main reason for this is for us the only polyfill we really want is for the @layers given the current Safari support [lack there of]

**Comment by anubra266** [comment]
@jimmymorris You can do this with postcss plugin
https://www.npmjs.com/package/@csstools/postcss-cascade-layers
